components/FlexibleForms/FlexibleFields.tsx

```diff
@@ -0,0 +1,134 @@
+import TextField from './TextField';
+import RadioField from './RadioField';
+import CheckboxField from './CheckboxField';
+import SelectField from './SelectField';
+import RepeaterField from './RepeaterField';
+import RepeaterGroupField from './RepeaterGroupField';
+import ComboboxField from './ComboboxField';
+import { FormikValues, FormikTouched, FormikErrors } from 'formik';
+import { Field } from 'data/flexibleForms/forms.types';
+
+interface Props {
+  values: FormikValues;
+  field: Field;
+  touched: FormikTouched<FormikValues>;
+  errors: FormikErrors<FormikValues>;
+}
+
+const FlexibleField = ({
+  values,
+  field,
+  touched,
+  errors,
+}: Props): React.ReactElement | null => {
+  if (field.condition && values[field.condition.id] !== field.condition.value)
```

Comment:
Could I have some clarification on the purpose of this if statement

Reply:
it makes conditional fields, that only show up based on a previous answer, work. for example:

- Do you take any medications? (yes/no radio)
- List your prescriptions (textarea)